server/mrt_converter/mrt_converter.go

```diff
@@ -112,3 +118,56 @@ func parseUpdate(collector string, h *mrt.MRTHeader, buf []byte) (*update, error
 		Attributes: translateAttrs(bgpUpdate.PathAttributes),
 	}, nil
 }
+
+// convert translates the bzip'ed MRT raw bytes into a BigQuery compatible
+// format and write to GCS.
+func convert(collector string, src []byte, dst io.Writer) error {
+	br := bzip2Reader(bytes.NewReader(src))
+	gr := gzip.NewWriter(dst)
+	defer gr.Close()
+
+	for {
+		buf := make([]byte, mrt.MRT_COMMON_HEADER_LEN)
+		_, err := io.ReadFull(br, buf)
+		if err == io.EOF {
+			break
+		} else if err != nil {
+			return fmt.Errorf("failed to read MRT header: %v", err)
+		}
+
+		h := &mrt.MRTHeader{}
+		err = h.DecodeFromBytes(buf)
+		if err != nil {
+			return fmt.Errorf("DecodeFromBytes: %v", err)
```

Comment:
some golang style nerd is going to say:
  "Hey, errors shouldnt' strart with capitalization nor end in punctuation"
  
  you might:
    "failed to decode header from bytes: %v"

instead :) or not, your call :)

Reply:
Oops, didn't notice it. 

Done.